chapter4.adoc

```diff
@@ -2,17 +2,17 @@
 [[Smmtt]]
 == `Smmtt`: Hart MTT Extension
 
-=== Smmtt[34, 46, 56]rw
+[[Smmtt-rw]]
+=== Smmtt[34 | 46 | 56]rw
 
 The `PPN` rooted structure for the MTT is shown below. The structure
 below shows a 56 bit physical address lookup; for lower physical address
-widths e.g. 46 bits, the `MTTL3` table is not applicable. In this mode of the
-Smmtt[34, 46, 56]rw, each page is associated with a read and a write access
-permission (2 bits). <<mtt-map-rw>> shows the Physical address (PA) being
-used to index into the `MTT` structure in memory to lookup access
-permissions for the supervisor domain specified in the `MTTL1` entry.
-Intermediate `MTTL3` and `MTTL2` entries are used to allow this structure to
-be sparsely populated.
+widths e.g. 46 bits, the `MTTL3` table is not applicable. Each
+physical-addressable page is associated with an access-permission encoding.
+<<mtt-map-rw>> shows the Physical address (PA) being used to index into the
+`MTT` structure in memory to lookup access permissions for the supervisor domain
+specified in the `MTTL1` entry. Intermediate `MTTL3` and `MTTL2` entries are
+used to allow this structure to be sparsely populated.
```

Comment:
Smmtt34 is clearly intended to support RV32. Howeve the L2 entries are 8 bytes wide which is wider than RV32 XLEN. Do we want to separate Smmtt34 to only have 4 byte L2 entries?

Reply:
will make all MTT L2 and L1 PTEs XLEN dependent